test/unit_test/engine/packaging/test_packaging_generator.py

```diff
@@ -67,6 +67,7 @@ def test_generate_zipfile_artifacts(mock_sys_getsizeof, save_as_external_data, m
         zip_ref.extractall(output_dir)
     assert (output_dir / "SampleCode").exists()
     assert (output_dir / "CandidateModels").exists()
+    assert (output_dir / "models_rank.json").exists()
```

Comment:
better load the json to verify the format is correct?

Reply:
Added to test.